README.md

```diff
@@ -53,7 +73,3 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Microverse: [microverse community](https://github.com/microverseinc)
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] All requirements implemented as requested by the previous reviewer. Please create a `MIT.md` license file and add the link you removed last time 👍. You can get the template from [this link](https://github.com/microverseinc/readme-template/blob/master/MIT.md)

Reply:
Hah thank @Rustamxon7 :+1: